HISTORY.md

```diff
@@ -5,6 +5,9 @@
 * Added EventListener::OnExternalFileIngested which will be called when IngestExternalFile() add a file successfully.
 * BackupEngine::Open and BackupEngineReadOnly::Open now always return error statuses matching those of the backup Env.
 
+### Bug fixes
+* When file copying is needed when creating checkpoints or bulk loading files, fsync the file after the file copying.
```

Comment:
It feels a bug to me. When we copy files when bulk loading files, we should make it durable. Otherwise, a machine crash after that may cause DB corruption.